dot/network/block_announce.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/libp2p/go-libp2p-core/peer"
 )
 
+var errInvalidRole = errors.New("invalid handshake role")
```

Comment:
nit should this be renamed to `errHandshakeRoleInvalid` since it's for handshakes? 🤔

Reply:
changed to `invalid handshake role` instead. `invalid handshake role` didn't really make sense, since role is a property of a node and not a handshake.